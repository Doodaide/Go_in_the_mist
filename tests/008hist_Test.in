place A1
place B2
place C3
place S10
place S19
place R18
place Q17
place P16
place O15
place N14
history
term