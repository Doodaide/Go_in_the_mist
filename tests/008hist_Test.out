A1B2C3S10S19R18Q17P16O15N14
